packages/client/modules/meeting/components/PokerTemplateList.tsx

```diff
@@ -63,8 +63,8 @@ const TabIcon = styled(Icon)({
   marginRight: 4
 })
 
-const containerStyle = {height: '100%'}
-const innerStyle = {width: '100%', height: '100%'}
+const containerStyle = {height: '85%'}
+const innerStyle = {width: '100%', height: '100%', maxWidth: 'calc(100vw - 48px)'}
```

Comment:
-1 the `maxWidth` change looks really good, but I think there's an underlying issue with the box model as child elements can be wider than their parent:

![child-width](https://user-images.githubusercontent.com/39854876/155584816-82c6252e-3a3f-41b7-b062-0ce015de8ce1.png)

The problem with using `calc(100vw - 48px)` here is other child elements that we add may encounter the same issue, so we could end up with lots of calcs in different places. Ideally, we can arrange the components so that the child width is 100% without it being wider than the parent.